easycv/core/sailfish/linear_test.py

```diff
@@ -0,0 +1,314 @@
+# Copyright 2019 Alibaba Inc. All Rights Reserved.
```

Comment:
move all test files to tests/core/sailfish , and rename filename as test_xxx.py

Reply:
done